server/datastore/mysql/migrations/tables/20240322145615_CreateTableNanoDDMRequests_test.go

```diff
@@ -0,0 +1,20 @@
+package tables
+
+import "testing"
+
+func TestUp_20240322145615(t *testing.T) {
+	db := applyUpToPrev(t)
+
+	//
+	// Insert data to test the migration
+	//
+	// ...
+
+	// Apply current migration.
+	applyNext(t, db)
+
+	//
+	// Check data, insert new entries, e.g. to verify migration is safe.
+	//
+	// ...
+}
```

Comment:
could you delete this file? `applyUpToPrev` and `applyNext` perform _all_ the previous migrations on a test db, which quickly adds up to the test time.

Reply:
Sure